design/vela-core/migrate-from-oam-runtime.md

```diff
@@ -0,0 +1,154 @@
+# How to upgrade to KubeVela
+
+What if I want to upgrade from [oam-kubernetes-runtime](https://github.com/crossplane/oam-kubernetes-runtime) to KubeVela? Here's a detailed guide!
+has already been merged into [vela-core](https://github.com/oam-dev/kubevela/pull/663) now. Here is the doc for users who want
+to migrate from OAM Runtime to use KubeVela.
+
+## For Users Who are Using OAM runtime as Standalone Controller
```

Comment:
We need to either capitalize all except prep. or only keep the first capitalized.
I would recommend the latter.

Reply:
sure